doc/theoretical_description_regression.rst

```diff
@@ -266,6 +314,10 @@ Key takeaways
   theoretical and practical coverages due to the larger widths of the prediction intervals.
   It is therefore advised to use them when conservative estimates are needed.
 
+- If the "exchangeability hypothesis" is not valid, typically for time series,
+  use EnbPI, and update the residuals (using :class:`partial_fit` method) each
```

Comment:
I would remove `partial_fit` method since we do not discuss the implementation but focus on the theoretical description.

Reply:
Done